pages/common/rails-routes.md

```diff
@@ -0,0 +1,20 @@
+# rails routes
+
+> List routes in a Rails application.
+> More information: <https://guides.rubyonrails.org/routing.html>.
+
+- List all routes:
+
+`rails routes`
+
+- List all routes in expanded format:
+
+`rails routes --expanded`
+
+- List routes partially matching URL helper method name, HTTP verb, or URL path:
+
+`rails routes -g {{ posts_path | GET | /posts }}`
```

Comment:
shouldn't it be `{{posts_path|GET|/posts}}`?

Reply:
You are correct. I've removed the additional spaces.